### ISSUE TITLE
Investigate alembic revision log message

### ISSUE THREAD

**Issue by mthaddon** [canonical/indico-operator#69]
### Bug Description

During the staging deployment of indico we saw the following log message: https://pastebin.ubuntu.com/p/kgjnSqJzYg/.

I believe this was because the indico pod was getting restarted in a loop, but at some previous point the tables had been created. Is this message simply a warning, or do we need to do something to follow up on this?

### To Reproduce

I suspect we could reproduce as follows:

1. Deploy in the usual manner
2. Kill a pod and watch it restart
3. Inspect the logs

I haven't confirmed this however.

### Environment

This is running on charmed kubernetes on OpenStack, using juju 2.9.32 and was deployed with revision 33 of the indico charm in the edge channel.

### Relevant log output

```shell
# See the pastebin above for more details, but:

2022-08-22T10:21:09.462Z [indico] Your database is not empty!
2022-08-22T10:21:09.462Z [indico] If you just added a new table/model, create an alembic revision instead!
2022-08-22T10:21:09.494Z [indico]
2022-08-22T10:21:09.495Z [indico] Tables in your database:
2022-08-22T10:21:09.495Z [indico]   * attachments.folders
```

### Additional context

N/A

**Comment by arturo-seijas** [comment]
That's expected behaviour. The script provided by Indico is being run on on every container startup and detects that those changed have already been applied. In any case, it would be nice to have it running only once per deployment.
